apps/rush-lib/src/cli/scriptActions/BulkScriptAction.ts

```diff
@@ -329,6 +406,18 @@ export class BulkScriptAction extends BaseScriptAction {
         ' For details, refer to the website article "Selecting subsets of projects".'
     });
 
+    this._watchParameter = this.defineFlagParameter({
```

Comment:
I think we should put this behind an experiment.

Reply:
If this is behind an experiment, should we make `runWatch` private?